pkg/metrics/metrics.go

```diff
@@ -39,6 +39,9 @@ const (
 	OperatorPortName = "http-metrics"
 	// CRPortName defines the custom resource specific metrics' port name used in the metrics Service.
 	CRPortName = "cr-metrics"
+
+	PortNumber   int32 = 8383
```

Comment:
Since `CRPortNumber` is used for `CRPortName`, would it be more clearer if we change `PortNumber` to `OperatorPortNumber` so it's more clearly for `OperatorPortName`.